src/functions/indGraphSparse.jl

```diff
@@ -1,40 +1,58 @@
-struct IndGraphSparse{T <: RealOrComplex} <: IndGraph
+struct IndGraphSparse{T <: RealOrComplex, Ti <: Int} <: IndGraph
```

Comment:
I think Int is an alias for Int64 or Int32 depending on the system, either restrict it to the same as in the definition of SparseMatrixCSC or don't restrict it at all.